ui/src/api/src/__mocks__/ApiClient.js

```diff
@@ -0,0 +1,35 @@
+import { getFacetsList } from "../../../../tests/mockutils";
+import FacetsResponse from "../model/FacetsResponse";
+
+export const mockCallApi = jest.fn(
+  (
+    path,
+    httpMethod,
+    pathParams,
+    queryParams,
+    headerParams,
+    formParams,
+    bodyParam,
+    authNames,
+    contentTypes,
+    accepts,
+    returnType,
+    callback
+  ) => {
+    if (path === "facets" && httpMethod === "GET") {
+      let getFacetsResponse = new FacetsResponse();
+      getFacetsResponse.count = 210;
+      getFacetsResponse.facets = getFacetsList(5, 6);
```

Comment:
getFacetsList(2, 2);

And anywhere else I might have missed

Reply:
Done